tools/system_libs.py

```diff
@@ -955,6 +955,8 @@ def get_libcall_files(self):
       path='system/lib/libc',
       filenames=['emscripten_memcpy.c', 'emscripten_memset.c',
                  'emscripten_scan_stack.c',
+                 'emscripten_get_heap_size.c',  # needed my malloc
+                 'sbrk.c',  # needed my malloc
```

Comment:
```suggestion
                 'emscripten_get_heap_size.c',  # needed by malloc
                 'sbrk.c',  # needed by malloc
```

Reply:
But why do malloc's needs need to appear in this particular place?

Reply:
This is the list of non-lto files.   Any symbol that is a libcall (i.e. can be generated during LTO), or a dependency of a libcall, cannot itself be an lto file.

Since malloc is a libcall, all its dependency needs to be excluded from LTO.

See the comment under `libmalloc`: https://github.com/emscripten-core/emscripten/blob/6d61e5dea2a5e093c19eb2d488e81df0a423b7e5/tools/system_libs.py#L1568-L1570

Reply:
I see, thanks!